java/version.txt

```diff
@@ -1 +1,2 @@
-version=2.0
+version=2.0-dev.1
```

Comment:
This file is generated from the version in the markdown spec, so we don't want to hand edit it :)